astroplan/plots/plots.py

```diff
@@ -0,0 +1,216 @@
+                 dark_plot=False):
+    """
+    TODO:
+        1) Timezones?
+        2) Limit airmass <= 3.
+        3) Plot lines instead of points.
+        4) Legend should not be transparent.
+        5) dark_plot option
+
+    Makes an airmass vs. time plot.
+
+    If an ax object is passed in, plots an additional airmass plot on top.
+    Otherwise, creates a new ax object with an airmass plot.
+
+    When a `Time` object with a single instance in time is used (e.g., if
+    len(time) = 1), a new `Time` object is created internally.  This new object
+    contains a sampling of times within a window centered on the input `Time`.
+
+    Parameters
+    ----------
+    target : `~astroplan.FixedTarget`
+        The celestial body of interest.
+
+    observer : `~astroplan.Observer`
+        The person, telescope, observatory, etc. doing the observing.
+
+    time : `~astropy.time.Time`
+        Used in generating data to plot.
+        Can be scalar (len(time)=1) or contain multiple times.
+
+    ax : `~matplotlib.axes.Axes` or None, optional.
+        The axes this plot will be added to. If none are passed in, new axes
+        will be created.
+
+    style_kwargs : Dictionary or Empty, optional.
+        A dictionary of keywords passed into `matplotlib.pyplot.plot_date`
+        to set plotting styles.
+
+    dark_plot : Boolean, optional.
+        Default is False.
+
+    Returns
+    -------
+    ax :  `~matplotlib.axes.Axes`
+        An axes object with added airmass vs. time plot.
+    """
+
+    # Set up plot axes and style if needed.
+    if ax is None:
+        ax = plt.gca()
+    if dark_plot is True:
+        raise NotImplementedError
+    if style_kwargs is None:
+        style_kwargs = {}
+    style_kwargs = dict(style_kwargs)
+    style_kwargs.setdefault('linestyle', '-')
+    style_kwargs.setdefault('color', 'b')
+    style_kwargs.setdefault('fmt', '-')
+
+    # Populate time window if needed.
+    if time.isscalar:
+        time = time + np.linspace(-12, 12, 100)*u.hour
+
+    # Calculate airmass
+    airmass = observer.altaz(time, target).secz
+
+    # Some checks & info for labels.
+    assert len(time) == len(airmass)
+
+    if hasattr(observer, 'name') is False:
+        observer_name = ''
+    else:
+        observer_name = observer.name
+
+    if hasattr(target, 'name') is False:
+        target_name = str('%.3f' % (random.random()))
+    else:
+        target_name = target.name
+    style_kwargs.setdefault('label', target_name)
+
+    observe_date = time[0].datetime.strftime('%Y-%m-%d')
+    observe_timezone = 'UTC'
+
+    # Plot data.
+    ax.plot_date(time.plot_date, airmass, **style_kwargs)
+    ax.figure.autofmt_xdate()
+    if plt.ylim()[1] > plt.ylim()[0]:
+        ax.invert_yaxis()
+
+    # Set labels, title, legend, etc.
+    ax.set_ylabel("Airmass")
+    ax.set_xlabel("Time - "+observe_timezone)
+    ax.set_title("Airmass vs Time | " + observer_name + " | " + observe_date +
```

Comment:
Not sure if we should assume a title or not -- my vote is no (don't set a title). What do others think? @astroplanners/astroplanners-contributors